client/asset/dcr/dcr.go

```diff
@@ -6699,12 +6699,7 @@ func (dcr *ExchangeWallet) handleTipChange(ctx context.Context, newTipHash *chai
 		dcr.cycleMixer()
 	}
 
-	var wg sync.WaitGroup
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
-		dcr.syncTxHistory(ctx, uint64(newTipHeight))
-	}()
+	go dcr.syncTxHistory(ctx, uint64(newTipHeight))
```

Comment:
How about using the `ExchangeWallet.wg` here instead so at least we don't have an unmonitored goroutine?

Reply:
Using the wg https://github.com/decred/dcrdex/compare/ec312e6851d5c63982c252ea92c99c8b4282b55f..9cc5f72d8810d4503859b46fd7899f4f6c110ff3